astroquery/mast/observations.py

```diff
@@ -546,11 +546,18 @@ def download_file(self, uri, local_path=None, base_url=None, cache=True, cloud_o
         url = None
 
         try:
-            if self._cloud_connection is not None and self._cloud_connection.is_supported(data_product):
-                try:
-                    self._cloud_connection.download_file(data_product, local_path, cache)
-                except Exception as ex:
-                    log.exception("Error pulling from S3 bucket: {}".format(ex))
+            if self._cloud_connection is not None:
```

Comment:
how is this related to this PR?

Reply:
This looks like an unrelated commit that was part of something else. I can make another commit to reverse these changes on my local branch.

Reply:
I'm going to revert these changes to what's in the `main` branch because it's following the same logic just written slightly differently, without the `fall_back` trigger and a few other things.